packages/slice-machine/src/legacy/lib/models/common/widgets/Image/Form.tsx

```diff
@@ -37,25 +37,21 @@ const FormFields = {
   },
   thumbnails: {
     validate: () =>
-      yup
-        .array()
-        .defined()
```

Comment:
The only actual change here is that I've removed `.defined()`. The rest is the auto-formatting.